mgr/src/cmds/volumes.cr

```diff
@@ -266,3 +266,24 @@ handler "volume.reset" do |args|
     puts "Volume #{volume_name} options reset"
   end
 end
+
+command "volume.rename", "Rename the Kadalu Storage volume" do |parser, _|
+  parser.banner = "Usage: kadalu volume rename POOL/VOLNAME POOL/NEW_VOLNAME [arguments]"
+end
+
+handler "volume.rename" do |args|
+  if args.pos_args.size < 2
+    puts "POOL/VOLNAME POOL/NEW_VOLNAME is required"
+    exit(0)
+  end
+
+  args.pool_name, volume_name = pool_and_volume_name(args.pos_args.size > 0 ? args.pos_args[0] : "")
+  existing_pool_name, new_volname = pool_and_volume_name(args.pos_args.size > 1 ? args.pos_args[1] : "")
```

Comment:
Will make use of `existing_pool_name` for volume transfer across pools as in `new_pool_name`.

Reply:
you can keep as `new_pool_name` itself.

Reply:
done